sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/models/SendEventsOptions.java

```diff
@@ -1,38 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-package com.azure.messaging.eventgrid.models;
```

Comment:
We should also remove the `package-info.java` in this `models` package as it'll now be empty.